tests/types/dateTime.spec.ts

```diff
@@ -0,0 +1,74 @@
+import {StrongDateTime, makeDateTime} from '../../src/types/dateTime';
+
```

Comment:
Update filename for project consistency to use all lowercase name with words separated by hyphens:

- [ ] Change filename from `dateTime.spec.ts` to `date-time.spec.ts`